src/python/grpcio/grpc/_cython/_cygrpc/aio/call.pyx.pxi

```diff
@@ -146,4 +168,15 @@ cdef class _AioCall:
             grpc_call_unref(call)
             gpr_free(ops)
 
-        return receive_message_operation.message()
+        if receive_status_on_client_operation.code() == GRPC_STATUS_OK:
+            return receive_message_operation.message()
+
+        class _RpcError(_RpcErrorBase, grpc.RpcError):
```

Comment:
is there a requirement on declaring this class on every `unary_unary` call? Can't we extract it outside?

Reply:
If this is declared in the main file, it doesn't compile property because it raises ``ImportError: 'grpc' does not have attribute RpcError`` (becaues it's a cython file calling a Python one), so it was done this way to have it inside the compiled code, so at running time the dependency will be availalble.

TL;DR the main constraing is ``grpc.RpcError`` being a Python class needed from a Cython one.